data.json

```diff
@@ -788,12 +788,14 @@
         "linux": {
             "root": "application-default-icon",
             "symlinks": [
+                "adriconf",
```

Comment:
I don't think we should use `application-default-icon` as a root icon for an actual application, has the potential to be misunderstood as a placeholder. Given that it's about graphics drivers the jockey icons might be more appropriate.